### ISSUE TITLE
Add "sentence" => normative change 1.4.8 Visual Presentation

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1603]
Success Criterion 1.4.8 Visual Presentation
https://www.w3.org/TR/WCAG22/#visual-presentation 

> For the visual presentation of blocks of text, a mechanism is available to achieve the following:

The SC is only applicable for "blocks of text".
The intent, rationale, user need is as well applicable for one (1) sentence, and this should be added in my oppinion.

For Success Criterion 2.5.5 Target Size we've added this already (in time...) and can be seen at the bullet:

> Inline: The target is in a sentence or block of text;

The SC will become:

> 1.4.8 Visual Presentation
> 
> For the visual presentation of a sentence or blocks of text, a mechanism is available to achieve the following:
